bundles/CoreBundle/src/DependencyInjection/Configuration.php

```diff
@@ -553,6 +553,10 @@ private function addAssetNode(ArrayNodeDefinition $rootNode): void
                                 ->defaultTrue()
                                 ->info('Process text for Asset documents (e.g. used by backend search).')
                             ->end()
+                            ->booleanNode('process_sanitizing')
```

Comment:
Actually "sanitizing" is maybe the wrong term for it, as we do not sanitize the PDF, but we're checking if it is potentially vulnerable 😬 
Maybe we find a bitter term for that.

Reply:
maybe simply call it `pdf_scanner`as it is scanning pdfs for Vulnerabilities so config would